scripts/deploy-federation.sh

```diff
@@ -144,10 +144,6 @@ kubectl apply --validate=false -f vendor/k8s.io/cluster-registry/cluster-registr
 
 # Enable available types
 for filename in ./config/federatedtypes/*.yaml; do
-  # The namespace controller is not required when the control plane is namespaced
-  if [[ "${NAMESPACED}" && "${filename}" == *namespace.yaml ]]; then
```

Comment:
Should we leave this in? It may help avoid confusing users thinking there's namespace push reconciliation support for namespace-scoped deployment without having to look at the logs.

Reply:
+1 to leave this in, I will also update the helm chart by adding some condition check to avoid creating namespace controller in namespaced scope federation controller.

Reply:
DRY - I don't think it's reasonable to require that every deployment mechanism know not to apply a federated type config for namespaces when deploying namespaced federation.  If/when user confusion becomes an issue we can revisit.